core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -693,7 +693,7 @@ class GroupCoordinator(val brokerId: Int,
   }
 
   private def onMemberFailure(group: GroupMetadata, member: MemberMetadata) {
-    debug(s"Member ${member.memberId} in group ${group.groupId} has failed")
+    info(s"Member ${member.memberId} in group ${group.groupId} has failed")
```

Comment:
This code path is also followed when we receive a LeaveGroup request. Do we want more verbose logging for that case as well? In any case, we should probably make the distinction clear in the log message.